android/src/main/java/com/facebook/reactnative/androidsdk/FBSDKPackage.java

```diff
@@ -43,7 +43,7 @@
                 new FBGraphRequestModule(reactContext),
                 new FBLoginManagerModule(reactContext, mActivityEventListener),
                 new FBMessageDialogModule(reactContext, mActivityEventListener),
-                new FBProfileModule(),
+                new FBProfileModule(reactContext),
                 new FBSettingsModule(),
```

Comment:
isn't FBSettingsModule going to have the same issue :thinking:

Reply:
Good question 🤔